src/styles/styles.js

```diff
@@ -29,33 +30,33 @@ const styles = StyleSheet.create({
   },
   artists: {
     fontSize: 20,
-    fontFamily: 'Palatino-BoldItalic',
+    fontFamily: 'Arial',
     fontWeight: 'bold',
-    color: 'salmon',
+    color: 'black',
   },
   track: {
     fontSize: 25,
-    fontFamily: 'Palatino-BoldItalic',
-    color: 'salmon',
+    fontFamily: 'Arial',
+    color: 'black',
   },
   image: {
     margin: 5,
     resizeMode: 'contain',
   },
   description: {
-    fontFamily: 'Palatino-BoldItalic',
+    fontFamily: 'Arial',
     fontSize: 15,
-    color: 'salmon',
+    color: 'black',
   },
   playButton: {
-    color: 'aliceblue',
+    color: 'grey',
     opacity: 0.6,
     fontSize: 70,
     alignSelf: 'center',
     marginTop: 40,
   },
   stopButton: {
-    color: 'lavenderblush',
+    color: 'grey',
```

Comment:
can't say i'm thrilled about these colour changes but thanks for taking care of the fonts